program/src/instruction.rs

```diff
@@ -629,11 +579,6 @@ accounts_struct! {
             is_signer: false,
             is_writable: false,
         },
-        pub validator_fee_st_sol_account {
-            is_signer: false,
-            is_writable: false,
-        },
-        const sysvar_rent = sysvar::rent::id(),
```

Comment:
In this case, because nobody external can call this instruction anyway, it _might_ be fine to edit it in-place, but it will still cause trouble. For example, if you use the `solido multisig show-transaction` command to inspect one of the transactions that added the initial Lido validators (to v1), then v2 of the `solido` program will no longer be able to parse that transaction.

We could accept this and say that you need `solido` 1.x to be able to inspect the transaction, but there is no way for people to figure that out from just looking at the multisig transaction's bytes.

It would still be nicer to not modify the existing transactions, and instead add a `AddValidatorV2` instruction. The implementation of `AddValidator` v1 can be replaced with just a panic that explains the instruction is deprecated, but we should keep the definition around.